tests/accordion/accordion.browser-test.jsx

```diff
@@ -0,0 +1,249 @@
+ * https://github.com/producthunt/chai-enzyme
+ */
+
+chai.use(chaiEnzyme());
+
+/* Re-usable demo component.
+ */
+
+const propTypes = {};
+
+const defaultProps = {};
+
+class AccordionExample extends React.Component {
+	constructor (props) {
+		super(props);
+		this.state = {
+			expandedPanels: {},
+			items: [
+				{
+					id: '1',
+					summary: 'The first item',
+					details:
+						'Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod ' +
+						'tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At v' +
+						'ero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, ' +
+						'no sea takimata sanctus est Lorem ipsum dolor sit amet.'
+				},
+				{
+					id: '2',
+					summary: 'The second item',
+					details: (
+						<div>
+							<p>Lorem ipsum dolor sit amet,</p>
+							<p>consetetur sadipscing elitr,</p>
+							<p>
+								sed diam nonumy eirmod tempor invidunt ut labore et dolore magna
+								aliquyam erat,
+							</p>
+							<p>sed diam voluptua.</p>
+							<p>At vero eos et accusam et justo duo dolores et ea rebum.</p>
+							<p>Stet clita kasd gubergren,</p>
+							<p>no sea takimata sanctus est Lorem ipsum dolor sit amet.</p>
+						</div>
+					)
+				},
+				{
+					id: '3',
+					summary: 'The third item',
+					details: (
+						<div>
+							<p>
+								Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed
+								diam nonumy eirmod tempor invidunt ut labore et dolore magna
+								aliquyam erat, sed diam voluptua.
+							</p>
+							<p>
+								At vero eos et accusam et justo duo dolores et ea rebum. Stet
+								clita kasd gubergren, no sea takimata sanctus est Lorem ipsum
+								dolor sit amet.
+							</p>
+						</div>
+					)
+				}
+			]
+		};
+	}
+
+	menuDropdown (selectedItem) {
+		return (
+			<Dropdown
+				align="right"
+				id="ButtonGroupExampleDropdown"
+				assistiveText="More Options"
+				buttonVariant="icon"
+				buttonClassName="slds-shrink-none"
+				iconName="down"
+				iconVariant="border-filled"
+				onSelect={(option) => {
+					if (option.label === 'delete') {
+						this.setState((state) => ({
+							...state,
+							items: state.items.filter((item) => item.id !== selectedItem.id)
+						}));
+					} else if (console) {
+						console.log('onSelect', event, option);
+					}
+				}}
+				openOn="click"
+				options={[
+					{
+						label: 'delete',
+						value: 'A0'
+					},
+					{
+						label: 'redo',
+						value: 'B0'
+					},
+					{
+						label: 'activate',
+						value: 'C0'
+					}
+				]}
+				iconSize="x-small"
+			/>
+		);
+	}
+
+	togglePanel (id) {
+		this.setState((state) => ({
+			...state,
+			expandedPanels: {
+				...state.expandedPanels,
+				[id]: !state.expandedPanels[id]
+			}
+		}));
+	}
+
+	render () {
+		return (
+			<IconSettings iconPath="/assets/icons">
+				<Accordion id="base-example-accordion" openOn="click">
```

Comment:
`openOn="click"` This seems out of place.

Reply:
yes, maybe this was just code left over from a copy/paste from another component? We should remove it in all the examples.